frontend/index.html

```diff
@@ -299,5 +299,12 @@ <h1>About</h1>
         />
       </symbol>
     </svg>
+    <script type="text/javascript">
+      // Avoid flicker of placeholder sample in editor if we have a saved session.
```

Comment:
I still got some bad flicker the first time I loaded the per-PR app deployment. At that point, nothing was cached and the page loaded with the default content, then cleared it for about 1 sec before it came back again. I can reproduce it with the dev tools, disabling the cache and setting the throttling to fast or slow 3G. A scrollbar also appears for a brief period too and then disappears when the editor takes over.

Reply:
I've added a last commit that rearranges the editorInitialisation and source code loading to avoid the flicker. Please let me know what you think.

Reply:
The flickering of the code has gone, but there is still a scrollbar that appears and disappears. It seems like it shows only 11 of the 12 lines initially, hence the scrollbar, and that goes away when the editor takes over.

I don't know if this only started in this PR or not, so feel free to leave it for later.

Reply:
ok, I will try to look into this. My scrollbars don't show up (only once I start scrolling). I think that's why I've missed it. I think this is new.

Reply:
ok, so I can reproduce this on Firefox on MacOS (not Chrome). I believe it already has been a thing previously: would you mind checking https://evy.dev with throttled network & no-cash? 
It is a bit different now though:

Previously the loading sequence was like this:

1. Load welcome source code with syntax colouring (uneditable) in index.html
2. Once editor and all JS is ready create editor with come source and syntax color it.
3. Load current example (depending on URL fragment).

We have no changed this to:

1. Load welcome source code with syntax colouring (uneditable) in index.html
    if there is anything in the sessionStorage load that without syntax colouring.
3. Load current example (depending on URL fragment) and load it straight into the 
editor.

Reply:
I have not managed to find any hints on how to remove the temporary scrollbar flicker on firefox :(

Reply:
I found this SO post: https://stackoverflow.com/questions/77453048/how-can-i-prevent-the-scrollbar-from-fading-in-firefox

It has a super simple index.html page demonstrating the issue:

```
<!doctype html>
<html lang="en">
  <head></head>
  <body>
    <div style="overflow: auto; max-width: 100px; max-height: 100px">
      dfkdjkfdfjkdkjfdfjkkkkkkkkkkkkkkkk dfdjkfdkjf dfjkdjfdf fdjk
    </div>
  </body>
</html>
```

In chrome on MacOS (for me) no scrollbars. On firefox: scrollbar shows and disappears.
Not really sure what to do about this.

Reply:
I can make scrollbars disappear altogether on firefox with `scrollbar-width: none;` as part of the style, but I think that's not great either.
I think the moment you try scrolling it is nice for scrollbars to show.

Reply:
Long story short, I might have found a workaround. I'm not sure if this makes things better... it's all added to yet another last commit. please let me know what you think.

Reply:
That all works better. No flickering. No scrollbar. Although I still only see up to "11" and then the "12" appears when the editor takes over. Not sure why, but I wouldn't worry about this.